### PR TITLE
Update danish.xml to 8.2.1 Fresh

### PR DESCRIPTION
**Pull request by Regmos**
Hi Don
Deleted my fork, created a new, edited the translation, committed and pulled a new request. Hope this works.
Kind regards 
Regmos

### CODE REVIEW

**Comment by donho** [comment]
@Regmos 
It should be OK now.
Thank you for your help.

File: PowerEditor/installer/nativeLang/danish.xml [hunk 1]
```diff
@@ -116,23 +116,23 @@ The comments are here for explanation, it's not necessary to translate them.
...
-                    <Item id="42066" name="Sortér linier som decimaltal med punktum (faldende)"/>
+                    <Item id="42059" name="Sortér linier alfabetisk (A > Å)"/>
+                    <Item id="42060" name="Sortér linier alfabetisk (Å > A)"/>
+                    <Item id="42080" name="Sortér linier alfabetisk (a > å, ignorér versaler)"/>
+                    <Item id="42081" name="Sortér linier alfabetisk (å > a, ignorér versaler)"/>
+                    <Item id="42061" name="Sortér linier som heltal (0 > ∞)"/>
+                    <Item id="42062" name="Sortér linier som heltal (∞ > 0)"/>
+                    <Item id="42063" name="Sortér linier som decimaltal med komma (0 > ∞)"/>
+                    <Item id="42064" name="Sortér linier som decimaltal med komma (∞ > 0)"/>
+                    <Item id="42065" name="Sortér linier som decimaltal med punktum (∞ > 0)"/>
```
Comment (Zemux1613) [inline PowerEditor/installer/nativeLang/danish.xml:134]: Is this line correkt?? I guess "0 > infinity" is the right here.

Comment (donho) [reply PowerEditor/installer/nativeLang/danish.xml:134]: The translation of "42062" - "42065" is "Sorting Lines" Ascending or Descending. there's no meaning of infinity (∞)

Comment (Zemux1613) [reply PowerEditor/installer/nativeLang/danish.xml:134]: Oke 👍

**Review by donho — CHANGES_REQUESTED** [review]
Remove ∞ which will confuse users

**Comment by Regmos** [comment]
Hi.
Replaced the infinit characters. 
Kind regards 
Regmos

**Review by donho — APPROVED** [review]

**Comment by dkgof** [comment]
In danish "linie" should be "linje", the other form is no longer valid since 2001:

https://sproget.dk/raad-og-regler/artikler-mv/svarbase/SV00005070

**Comment by donho** [comment]
> In danish "linie" should be "linje", the other form is no longer valid since 2001:

@Regmos I will merge your PR now.
But could you consider the point that @dkgof mentioned above for the next PR?

**Comment by donho** [comment]
@Regmos 
I've got some  errors again by merging your PR:
```
$ git checkout -b Regmos-master master
fatal: A branch named 'Regmos-master' already exists.
```

Here is the guideline of contribution regarding related issue:
`Create a new branch for each PR. Make sure your branch name wasn't used before - you can add date (for example patch3_20200528) to ensure its uniqueness.`
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests

I'll fix it manually this time, but please apply the guideline for your next PR.

**Pull request closed without merging**